control-plane/connect-inject/controllers/endpointsv2/endpoints_controller.go

```diff
@@ -493,7 +498,7 @@ func getEffectiveTargetPort(targetPort intstr.IntOrString, prefixedPods selector
 
 	// If still no match for the target port, fall back to string-ifying the target port name, which
 	// is what the PodController will do when converting unnamed ContainerPorts to Workload ports.
-	return targetPort.String()
+	return "cslport-" + targetPort.String()
```

Comment:
This feels like it belongs in some constants package.

Reply:
Ah yep will refactor